plugin/src-mill0.10/io/kipp/mill/scip/Scip.scala

```diff
@@ -311,30 +266,10 @@ object Scip extends ExternalModule {
       createFolders = true
     )
 
-    val classPathEntries =
-      classpath.flatMap(project => ClasspathEntry.fromPom(project.toNIO))
-
-    log.info(s"Including ${classPathEntries.size} classpath entries")
-
-    val options = new ScipSemanticdbOptions(
-      List(dest).map(_.toNIO).asJava,
-      scipFile.toNIO,
-      workspace.toNIO,
-      reporter,
-      toolInfo,
-      "java",
-      outputFormat,
-      true, // parallel -- this is fine
-      classPathEntries
-        .map(_.toPackageInformation)
-        .asJava,
-      "", // BuildKind here is fine being ""
-      true, // emit inverse releationships,
-      false, // we want to fail with mill if no documents have been indexed
-      false // we're only dealing with jar files here
-    )
-
-    ScipSemanticdb.run(options)
+    val exit = ScipJava.app.run(List("index-semanticdb", "--cwd", workspace.toString, "--output", scipFile.toString, dest.toString))
```

Comment:
It looks like the classpath entries aren't being included here?

Reply:
The classpath is automatically inferred from `javacopts.txt`, which is written to the targetroot